src/components/Hero/Abilities.jsx

```diff
@@ -23,23 +23,18 @@ const AbilityItem = styled.div`
   padding-right: 4px;
 `;
 
-const renderAbilities = (abilities) => {
-  const mappedAbilities = abilities.map((ability, key) => {
-    const mappedAbility = (
-      <AbilityItem key={key}>
-        <Ability {...ability} abilityID={key} />
-      </AbilityItem>
-    );
-
-    return mappedAbility;
-  });
-
-  return mappedAbilities;
-};
+const renderAbilities = abilities => abilities.map((ability, key) => (
+  <AbilityItem key={key}>
+    <Ability {...ability} abilityID={key} />
+  </AbilityItem>
+));
 
 const Abilities = ({ hero, abilities, heroAbilities }) => {
-  const filterAbilities = abilities => abilities.filter(ability => (ability !== 'generic_hidden'));
-  const mapAbilities = heroAbilities => heroAbilities.map(ability => abilities[ability]);
+  const filterAbilities = toFilterAbs => toFilterAbs.filter(ability => (ability !== 'generic_hidden'));
+
+  function mapAbilities(toMapHeroAbs) {
```

Comment:
was there a reason to change this to a function declaration?

Reply:
Just for readability, I can revert it.

Reply:
Also, I think it was something weird that my IDE did and I didn't notice.